frontend/src/pages/projects/screens/detail/storage/StorageTableRow.tsx

```diff
@@ -79,6 +107,47 @@ const StorageTableRow: React.FC<StorageTableRowProps> = ({
           </Flex>
           <Text>{getDescriptionFromK8sResource(obj)}</Text>
         </Td>
+
+        {isStorageClassesAvailable && (
+          <Td dataLabel="Storage class">
+            <TextContent>
+              <Text>
+                {obj.spec.storageClassName}{' '}
```

Comment:
this should fall back to the metadata name if the storage class config is undefined